environment.yml

```diff
@@ -8,7 +8,12 @@ dependencies:
   - pip
   - geopandas
   - ipywidgets
+  - libpysal
+  - matplotlib
   - jupyter
   - jupyterlab
   - nbconvert
-  - brewer2mpl
+  - palettable
+  - descartes
+  - pip:
+    - git+https://github.com/sjsrey/mapclassify.git@binder
```

Comment:
* Should this be `pysal/mapclassify.git@binder`?
* Where is the binder being launched from? Shouldn't there be a badge or something in the `README.md`?

Reply:
Yeah. I needed to get it working first on my fork. 
Once this is merged we can get the badge for upstream.

Reply:
* binder launches successfully 
* notebooks run successfully:
  * `01_maximum_breaks.ipynb`
  * `02_legends.ipynb`
  * `03_choropleth.ipynb`
  * `04_pooled.ipynb`
* notebooks with errors:
  * `05_Greedy_coloring.ipynb`

The following traceback is produced from cell 3 in `05_Greedy_coloring.ipynb`; looks like we need to add `networkx` into `environment.yml` (and also `requirements.txt` and `mapclassify.yml`?):

```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
/srv/conda/envs/notebook/lib/python3.6/site-packages/mapclassify/greedy.py in greedy(gdf, strategy, balance, min_colors, sw, min_distance, silence_warnings, interchange)
    287     try:
--> 288         import networkx as nx
    289     except ImportError:

ModuleNotFoundError: No module named 'networkx'

During handling of the above exception, another exception occurred:

ImportError                               Traceback (most recent call last)
<ipython-input-3-6603cfcd9243> in <module>
----> 1 africa['greedy_default'] = greedy(africa)
      2 africa['greedy_default'].head(5)

/srv/conda/envs/notebook/lib/python3.6/site-packages/mapclassify/greedy.py in greedy(gdf, strategy, balance, min_colors, sw, min_distance, silence_warnings, interchange)
    288         import networkx as nx
    289     except ImportError:
--> 290         raise ImportError("The 'networkx' package is required.")
    291     try:
    292         import pandas as pd

ImportError: The 'networkx' package is required.
```

Reply:
Need to add `seaborn`, too (cell 9 in `05_Greedy_coloring.ipynb`):

```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-9-2edc5b0bfc23> in <module>
      2 import numpy as np
      3 from time import time
----> 4 import seaborn as sns
      5 import matplotlib.pyplot as plt
      6 

ModuleNotFoundError: No module named 'seaborn'
```

Reply:
`05_Greedy_coloring.ipynb` now runs successfully.